src/plotloader/ScatterPlot.js

```diff
@@ -131,8 +131,8 @@ const highchartsBaseConfig = {
         states: {
           select: {
             style: {
-              fontWeight: 'normal',
-              color: 'black'
+              fontWeight: `normal`,
+              color: `black`
```

Comment:
These small changes fill my heart with joy. :smile: Glad to see you getting the hang of it!